<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileOptDialog</class>
 <widget class="QWidget" name="FileOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="lbFilePath">
     <property name="text">
      <string>File Path</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="sBTimeStart">
     <property name="toolTip">
      <string>Start time</string>
     </property>
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lEFilePath">
     <property name="toolTip">
      <string>File path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cBTimeTag">
     <property name="toolTip">
      <string>Generate time tag file</string>
     </property>
     <property name="text">
      <string>Time tag file</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cBTimeSpeed">
     <property name="toolTip">
      <string>File read speed</string>
     </property>
     <item>
      <property name="text">
       <string>x0.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x0.2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>x10</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lbPlus">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cBPathEnable">
     <property name="toolTip">
      <string>Specify file path</string>
     </property>
     <property name="text">
      <string>File Path</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbSwapInterval">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Swap Interval</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBSwapInterval">
        <property name="toolTip">
         <string>Swap file periodically. User placeholder keyword to avoid overwriting (see ?-button).</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.25 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.5 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12 h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 h</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cB64Bit">
        <property name="toolTip">
         <string>64 bit format for time tag file</string>
        </property>
        <property name="text">
         <string>64bit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnKey">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Show placeholder keys</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icon/resources.qrc">
          <normaloff>:/buttons/key</normaloff>:/buttons/key</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBSwapInterval</tabstop>
  <tabstop>btnKey</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../icon/resources.qrc"/>
 </resources>
 <connections/>
</ui>
